src/templates/base.html

```diff
@@ -177,6 +177,7 @@ <h4 class="ui inverted header">Codalab v2</h4>
     <script>
         {# From `common_settings` context processor #}
         STORAGE_TYPE = "{{ STORAGE_TYPE }}"
+        CURRENT_DATE_TIME = "{% now "c" %}"
```

Comment:
@ckcollab Do we want/need this accessible from every single page? or just the comp detail page? could maybe pass this in opts on the html page w/ the comp-detail riot tag?

Reply:
I think it could be useful, yeah, what do you think?